src/server/serverSetup.js

```diff
@@ -114,18 +119,35 @@ fluid.defaults("sjrk.storyTelling.server", {
                             }
                         }
                     },
-                    testBinaries: {
+                    testData: {
                         type: "kettle.middleware.static",
                         options: {
-                            "root": "./tests/binaries"
+                            "root": "./tests/testData"
                         }
                     },
                     ui: {
                         type: "kettle.middleware.static",
                         options: {
-                            "root": "./ui",
+                            "root": "./src/ui"
+                        }
+                    },
+                    // the custom theme for the site, loaded in "on top" of base
+                    currentTheme: {
+                        type: "kettle.middleware.static",
+                        options: {
+                            root: "@expand:sjrk.storyTelling.server.getCustomThemeFolder({server}.options.globalConfig.theme, ./themes/%theme)",
```

Comment:
The base theme directory `./themes` can be defined as a configurable component option.

Reply:
Do you mean to suggest that I could configure the root directory for the baseTheme middleware? In that case I intentionally chose not to do this, in order to avoid confusion on the part of someone else who would like to implement their own custom theme. I could make it a configurable option and simply not pull that from `sjrk.storyTelling.server.config.json5`, but I thought this was a reasonable implementation, since I only wanted to change it for node tests.

Alternatively, are you suggesting that in this particular case that the second argument for `getCustomThemeFolder()` should be an option instead of hardcoded here?

Reply:
Sorry about the confusion. The "configurable" I meant was to define the base theme dir "./themes" as a component option rather than via `sjrk.storyTelling.server.config.json5` for end users. I think this flexibility will make it easier for reusing or configuring the component as well as writing tests as you said.